operator/cmd/mesh/testdata/manifest-generate/data-snapshot/profiles/default.yaml

```diff
@@ -594,7 +595,7 @@ spec:
         jaegerURL:
         jaegerInClusterURL: http://tracing/jaeger
         auth:
-          strategy: ""
+          strategy: login
```

Comment:
@jmazzitelli I am not sure this is correct, although I am going to merge this PR for other reasons. If you believe the strategy should remain (anonymous? with `""`, please submit a new PR.

Reply:
This value should be "login". In fact, setting it to `""` will cause problems (Kiali doesn't have a default auth strategy - you must tell Kiali how you want to authenticate users).  You can see in the istioctl manifest chart that it is "login" here - we want this: https://github.com/istio/istio/blob/master/manifests/charts/istio-telemetry/kiali/values.yaml#L42

Reply:
The value being "" is what's wrong with 1.6 at the moment where no one can login to the Kiali dashboard (getting started page in docs has you login to dashboard and one can't). Setting this to login is supposed to fix that from what I understand.